README.md

```diff
@@ -24,12 +24,14 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Libraries and Frameworks
 - CSS
   - [Hexa](https://github.com/amine1107/Hexa) A simple and lightweight CSS animation library
+  - [Lighter](https://github.com/amine1107/lighter) Lighter is a flexible Css micro framework based on the visual aspect, the framework offers you a clean and easy work.
 - JavaScript
   - [Arli](https://github.com/elkebirmed/arli) - JavaScript tools for an Arabic development
   - [jQuery Ajax Native](https://github.com/acigna/jquery-ajax-native) - jQuery Plugin that adds XHR2 responseType support
   - [Nightly.js](https://github.com/Fcmam5/nightly.js) - A zero dependency JavaScript library for enabling the Dark mode in your UI
   - [react-graphql-redux](https://github.com/youknowriad/react-graphql-redux) - use GraphQL to query your Redux store
   - [Rungen](https://github.com/youknowriad/rungen) - A generic runtime around async flow in javascript
+  - [Siga](https://github.com/amine1107/siga) - JavaScript Library to make development in javascript easier. 
```

Comment:
This one needs a license and maybe a little bit better README to explain how to use it (maybe just a link to the docs). ;)

Reply:
the documentation is on github page,and I'm working on readme